Cloud Application Manager/Administering Your Organization/manage-assets-monitor-usage.md

```diff
@@ -50,53 +56,68 @@ To see more info about the options in this section you can refer to these pages:
 * [Monitoring](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/cammonitoringui)
 * [Analytics](https://www.ctl.io/knowledge-base/cloud-application-manager/analytics)
 * [Network Exchange](https://www.ctl.io/knowledge-base/network/network-exchange-getting-started-guide)
-* Show Unregistered Resources: By enabling this option in the Settings, you'll be able to see the unregistered instances under the CAM/ALM Instances menu.
+* Show Unregistered Resources: By enabling this option, you'll be able to see the unregistered instances under the CAM/ALM Instances menu.
 
-![Organization Settings - Details](../../images/cloud-application-manager/admin-assets1.png)
+![Organization Settings - Features](../../images/cloud-application-manager/admin-assets1-2.png)
 
 ### Manage Administrators
 
-Under **Administrators**, you can see a list of all administrators of the organization. For more details, visit [Give admin access to Cloud Application Manager](admin-access.md)
+Under **Administrators**, you can see a list of all administrators of the organization, as well as a search box for looking for a concrete user if there are at least ten administrators. For more details, visit [Give admin access to Cloud Application Manager](admin-access.md)
 
 ![Organization Settings - Manage Admins](../../images/cloud-application-manager/admin-assets2.png)
 
 ### Manage Clouds
 
-All clouds available in the organization are listed under **Clouds** section. Visit [Enable Access to Cloud Providers](provider-access.md).
+All clouds available in the organization are listed under **Clouds** section. Visit [Enable Access to Cloud Providers](provider-access.md) for more information.
 
 ![Organization Settings - Manage Clouds](../../images/cloud-application-manager/admin-assets3.png)
 
 ### Manage Tags
 
-Take a look at all tags in the organization under **Tags** in the admin console. More information in [Tag Instances](resource-tags.md)
+Take a look at all tags in the organization under **Tags** in the admin console. More information in [Tag Instances](resource-tags.md).
 
 ![Organization Settings - Manage Tags](../../images/cloud-application-manager/admin-assets4.png)
 
 ### Manage Webhooks
 
-View all webhooks available in the organization under **Webhooks**. In order to add a new webhook, click **Add** and write the URL.
+View all webhooks available in the organization under **Webhooks**. In order to add a new webhook, click **Add** and fill in the URL.
```

Comment:
Is there any other place where webhooks are explained in more detail? If yes, add link, otherwise, please add some more explanation here.